CRM/Core/Resources/Common.php

```diff
@@ -204,6 +204,10 @@ protected static function coreResourceList($region) {
       "js/wysiwyg/crm.wysiwyg.js",
     ];
 
+    $basePage = $settings->get('wpBasePage');
+    if (empty($basePage)) {
+      $basePage = 'civicrm';
+    }
```

Comment:
Why is this setting being applied unconditionally? Wouldn't this depend on whether we are on a front-end or back-end page?

Reply:
Because that's the default unless it has been changed. The "basePage" is only actually used on frontend pages.